docs/docs-content/vm-management/architecture.md

```diff
@@ -0,0 +1,313 @@
+| **Storage Adapters** | 2 x 16 Gbps FC                                     | 2 x 16 Gbps FC                               |                                                                                  |
+| **Server**           | Local disk for the OS boot (SAN boot is supported) | Local disk for the OS boot                   | Boot from SAN requires special consideration due to the multipath configuration. |
+
+Typically, the cluster control plane nodes do not run any VMO workloads. As a result, they can have lighter hardware
+specifications. For example, a server with 4 cores and 8 GB RAM is sufficient for a minimum-specification control plane
+node.
+
+You can increase the hardware specifications based on the total number of control plane and worker nodes you want in the
+cluster. Refer to the following table for guidance on control plane node sizing. Note that these recommendations assume
+that each cluster has at least three control plane nodes.
+
+:::info
+
+A Palette VMO cluster should consist of at least 10 worker nodes to ensure sufficient resource distribution. The
+[Portworx Sharedv4 service pod anti-hyperconvergence](https://docs.portworx.com/portworx-enterprise/concepts/shared-volumes#sharedv4-service-pod-anti-hyperconvergence)
+setting, which is necessary for seamless storage failover, requires two nodes to host its persistent storage, and these
+two nodes cannot run any VMs. As a result, your VMO cluster needs to have enough nodes so that there are always at least
+two nodes available for the storage failover workload.
+
+:::
+
+| Worker Nodes | Namespaces | CPU Cores | Memory (GB) |
+| ------------ | ---------- | --------- | ----------- |
+| 10           | 100        | 4         | 8           |
+| 25           | 500        | 4         | 16          |
+| 100          | 1000       | 8         | 32          |
+| 250          | 2000       | 16        | 64          |
+| 500          | 4000       | 32        | 128         |
+
+:::tip
+
+For Pure Storage FlashArray, either the
+[FlashArray//C](https://www.purestorage.com/products/unified-block-file-storage/flasharray-c.html#specifications) or
+[FlashArray//X](https://www.purestorage.com/products/unified-block-file-storage/flasharray-x.html#specifications) models
+are compatible with Portworx and suitable for Palette VMO.
+
+:::
+
+## Software Resources
+
+Refer to the following table for the versions of software components we recommend using with Palette VMO.
+
+:::info
+
+You can make substitutions in our software recommendations to better fit your specific use case. For more information on
+how you can adapt the recommended software resources, get in touch with our
+[Customer Support](https://spectrocloud.atlassian.net/servicedesk/customer/portals).
+
+:::
+
+| Component                             | Software                                               | Recommended Version                                                                                               |
```

Comment:
Do we have to list specific versions? This will require us to maintain the table, so I wonder if we can do without it or may state it as minimum supported versions.